http-gateway/web/src/containers/app/app.js

```diff
@@ -0,0 +1,50 @@
+import { hot } from 'react-hot-loader/root'
+import { useAuth0 } from '@auth0/auth0-react'
+import Container from 'react-bootstrap/Container'
+import classNames from 'classnames'
+import { Router } from 'react-router-dom'
+
+import { LeftPanel } from '@/components/left-panel'
+import { Menu } from '@/components/menu'
+import { StatusBar } from '@/components/status-bar'
+import { Footer } from '@/components/footer'
+import { Routes } from '@/routes'
+import { useLocalStorage } from '@/common/hooks'
+import { history } from '@/store/history'
+import './app.scss'
+
+const App = () => {
+  const { isLoading, isAuthenticated, error, loginWithRedirect } = useAuth0()
+  const [collapsed, setCollapsed] = useLocalStorage('leftPanelCollapsed', false)
+
+  if (error) {
+    return <div>Oops... {error.message}</div>
```

Comment:
should be translated.

Reply:
It's a placeholder, will be replaced with a full error page with translated messages.